articles/governance/policy/concepts/definition-structure.md

```diff
@@ -307,6 +310,10 @@ letter, `.` to match any character, and any other character to match that actual
 **match** and **notMatch** are case-sensitive. Case-insensitive alternatives are available in
 **matchInsensitively** and **notMatchInsensitively**. For examples, see [Allow several name patterns](../samples/allow-multiple-name-patterns.md).
 
+When working with **\[\*\] alias** array field values, note that that every element of the
+array is evaluated individually, with logical AND between them. For details see
+[Evaluating the \[\*\] alias](../how-to/author-policies-for-arrays.md#evaluating-the--alias).
+
```

Comment:
This section should be removed.  Aliases and the [*] alias aren't introduced yet, so this could be confusing.  This information is already in both the count and alias area of the same page.  Please remove this addition.

Reply:
@DCtheGeek What does it mean "Aliases aren't introduced yet"?
I see they are used in the following **Count** section and there is actually a section named **Aliases** afterwards, which I think is a little too late - the fact that Policy works by comparing a `field` value to a defined value, and that the `field` can be single value or an array should be introduced at the very beginning. Moreover, who from the new readers like me would expect that aliases actually mean arrays?

Instead of removal, would you see a better place to fit it?

Reply:
You've persuaded me. Can you remove the "note that that" and the comma after individually.  That'll clean up the sentence and make it more active voice.  With that change, I'll accept the PR.  Thanks again for the contrib and the discussion!

Reply:
Haha, didn't notice that that. :)
Before we close this one, do you agree it would be good to mention the concept of comparison and single vs array fields somewhere in the beginning? I can't really decide where exactly and in what words to say that so I just committed something to get your feedback. Maybe it's nonsense, maybe it's poor wording, but I consider this actually the most important message of the entire article.